README.md

```diff
@@ -168,6 +173,19 @@ Don't put `[` or `]` in your SSID.
 Also, the length of each SSID should be restricted between 0 and 32.
 The default value of `ssid_list` is `[MyHomeWiFi]` if it's not specified explicitly by user.
 
+Optional, you can let `owl0sink` to be an AP. First you need a `hostapd.conf`:
```

Comment:
Could `hostapd.conf` file just be put into `scripts` folder, and write the verified test case for this change.
It will be much better to write a lot of documents in README.md

Reply:
Please improve your English writing.

Reply:
Ok. I wrote the verified test for this change. And I reduced the description of README.md.